compiler/src/utils/config.rei

```diff
@@ -193,3 +193,7 @@ type implicit_opens =
   | Gc_mod;
 
 let get_implicit_opens: unit => list(implicit_opens);
+
+/** Formatter rewrites file in place */
+
+let format_in_place: ref(bool);
```

Comment:
@ospencer So do we want this moved into `grainformat.re`? Or is that left to #917?

Reply:
Yeah, let's get that moved before this is merged.

Reply:
@marcusroberts If you could move this flag out of `config.re` and into `grainformat.re` then that'd be :100:

Reply:
That's done